index.html

```diff
@@ -9,31 +9,51 @@
     <script src="https://kit.fontawesome.com/091ffb7972.js" crossorigin="anonymous"></script>
 </head>
 <body>
+  <div class="background-desktop">
```

Comment:
- [ ] Dear @ijosue16, Good job with the overall design. Yet, your background image seems to be acting differently than the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design on mobile and some desktop screen sizes.  Let's fix this to be exact with the Figma design.
- > On my screenshot you can find which screen size your design is acting the way I post it (all the mobile screenshots are on `375px`).
 > You can use `background-size: 100% auto`, which allows the height to grow automatically and the width always stays 100%, and also `background-position: top` just to fix it on the top.

## Figma backgrounds 
|   |   |   
|---|---|
![figma-background](https://user-images.githubusercontent.com/69077061/200372684-b2b9052f-8839-492a-bec0-a9c866b9c167.PNG)  | ![figma-background-2](https://user-images.githubusercontent.com/69077061/200372692-1cf7ab0e-c5bc-4e18-84e1-22a1cba3de98.PNG) |

## Your backgrounds
|   |   |   
|---|---|
![your-background](https://user-images.githubusercontent.com/69077061/200372784-f96d743a-3de5-4585-9b99-ce2fac0263de.png) |  ![your-background-2](https://user-images.githubusercontent.com/69077061/200372793-6eaadf6f-5753-445e-bdda-a04c0e80c76d.PNG)